README.md

```diff
@@ -94,9 +94,35 @@ Browse available modules on
 Standard library modules come from
 [micropython-lib](https://github.com/micropython/micropython-lib) project.
 
-(*) Debian/Ubuntu/Mint derivative Linux distros will require build-essentials,
-libffi-dev and pkg-config packages installed. If you have problems with some
-dependencies, they can be disabled in unix/mpconfigport.mk .
+External dependencies
+---------------------
+
+Building Unix version requires some dependencies installed. For
+Debian/Ubuntu/Mint derivative Linux distros, install `build-essentials`
+(includes toolchain and make), `libffi-dev`, and `pkg-config` packages.
+
+Other dependencies can be built together with MicroPython. Oftentimes,
+you need to do this to enable extra features or capabilities. To build
+these additional dependencies, first fetch git submodules for them:
+
+    $ git submodule update --init
+
+Use this same command to get the latest versions of dependencies, as
+they are updated from time to time. After that:
+
+    $ make deplibs
```

Comment:
Having the submodule update command makes you think that you should be in the root dir to execute that, and then so you'd think to execute make deplibs in the root dir.

And having this section at the same level as the previous unix section, instead of a subsection, makes it fell disconnected from unix.

Anyway, maybe just say: "... are update from time to time.  After that, in unix/ dir, execute: